package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-edit-ui",
-  "version": "0.6.0",
+  "version": "0.6.1",
```

Comment:
After you change this, you need to do `npm i` so the package-lock file is also updated (or you can cheat and update it manually).

Reply:
ok